src/main/frontend/src/components/WordReviewForm.js

```diff
@@ -73,8 +72,8 @@ const WordReviewForm = props => {
       })
         props.setReviewStatus("Thanks for your comment!")
       })
-      .then(word => {
-        history.pushState(`/words/${id}`)
+      .then( ()=>{
+        window.location.reload()
```

Comment:
You can actually use a functionality courtesy of react-router to redirect!
`import { Redirect } from 'react-router-dom'`

You'll first want to set up state to determine if it should be redirecting:
`const [shouldRedirect, setShouldRedirect] = useState(false)`

In your `.then`, you'll call `setShouldRedirect(true)`.

Finally, above your `return` on line 85, add a conditional return:

```
if (shouldRedirect) { // this is new!
    return <Redirect to=`/words/${id}` />
  }
```

Let me know if you have questions on this!

Docs: https://reactrouter.com/web/api/Redirect
A reference here, although it uses class syntax to do the same as I described above:
https://ui.dev/react-router-v4-programmatically-navigate/